src/App.vue

```diff
@@ -52,27 +52,32 @@ export default {
           The order in the object is the display order
         */
         planning: {
-          time: DEVELOPEMENT_MODE ? 5000 : 1000 * 30,
+          time: DEVELOPEMENT_MODE ? 5000 : returnTimeForPlanning(),
```

Comment:
pas appeler la fonction dans data mais dans changeView() (+ il  manque le `this.`)

Ici le code est éxécuté quand App est rendu donc au début, alors que le code de changeView est éxécuté à intervalle régulier